src/components/App/index.js

```diff
@@ -60,23 +61,23 @@ export function App({
         onChange={setUserIncome}
       />
       # Your Housing Type
-      <select value={userUnitSize} onChange={event => setUserUnitSize(event.target.value)}>
-        {UNIT_SIZES_RENT.map(size => (
-          <option value={size} key={size}>{size}</option>
-        ))}
-      </select>
+      <Dropdown
+        value={userUnitSize}
+        onChange={event => setUserUnitSize(event)}
+        options={UNIT_SIZES_RENT}
+      />
       # Others Housing Type
-      <select value={otherUnitSize} onChange={event => setOtherUnitSize(event.target.value)}>
-        {UNIT_SIZES_AFFORDABILITY.map(size => (
-          <option value={size} key={size}>{size}</option>
-        ))}
-      </select>
+      <Dropdown
+        value={otherUnitSize}
+        onChange={event => setOtherUnitSize(event)}
+        options={UNIT_SIZES_AFFORDABILITY}
+      />
```

Comment:
After #39 there will be just one dropdown for both . . . not sure how they'll be arranged but we'll figure that out later.